proto/01-protoc-base/install.sh

```diff
@@ -33,10 +33,19 @@ js_ver=3.21.2
 wget -q "https://github.com/protocolbuffers/protobuf-javascript/releases/download/v${js_ver}/protobuf-javascript-${js_ver}-linux-x86_64.zip"
 sha256sum --check "${__dir}/checksums.txt"
 
+doc_plugin_ver=1.5.1
+wget -q "https://github.com/pseudomuto/protoc-gen-doc/releases/download/v${doc_plugin_ver}/protoc-gen-doc_${doc_plugin_ver}_linux_amd64.tar.gz"
+
 pbuf_zip=protoc-${pbuf_ver}-linux-x86_64.zip
 unzip "$pbuf_zip" -d /opt/protoc
 rm "$pbuf_zip"
 
+tar -xzf "protoc-gen-doc_${doc_plugin_ver}_linux_amd64.tar.gz" -C /usr/local/bin
+rm "protoc-gen-doc_${doc_plugin_ver}_linux_amd64.tar.gz"
+
+# move protoc to PATH so it can be used in doc generation
+mv /opt/protoc/bin/protoc /usr/local/bin/
```

Comment:
I don't think this sort of change should be necessary. On the core side, we invoke directly:

```sh
  /opt/protoc/bin/protoc \
    --plugin=protoc-gen-grpc=/opt/java/bin/protoc-gen-grpc-java \
    ...
```

I'm assuming that protoc-gen-doc can be invoked similarly?

According to https://github.com/pseudomuto/protoc-gen-doc?tab=readme-ov-file#using-a-precompiled-binary, we should be able to do:

```sh
/opt/protoc/bin/protoc \
  --plugin=protoc-gen-doc=./protoc-gen-doc \
  ...
```